runtime/integration-tests/src/amplitude_tests.rs

```diff
@@ -12,95 +11,145 @@ use crate::{
 };
 
 use frame_support::assert_ok;
-use statemine_runtime as kusama_asset_hub_runtime;
+use asset_hub_kusama_runtime;
+use integration_tests_common::{
+	constants::{kusama, asset_hub_kusama},
+};
+
+use crate::genesis::{genesis_gen, genesis_sibling};
 use xcm::latest::NetworkId;
-use xcm_emulator::{decl_test_network, decl_test_parachain, decl_test_relay_chain, TestExt};
+use xcm_emulator::{decl_test_networks, decl_test_parachains, decl_test_relay_chains, DefaultMessageProcessor};
+use frame_support::traits::OnInitialize;
+
 
 // Native fee expected for each token according to the `fee_per_second` values defined in the mock
 const NATIVE_FEE_WHEN_TRANSFER_TO_PARACHAIN: polkadot_core_primitives::Balance = 4000000000;
-const KSM_FEE_WHEN_TRANSFER_TO_PARACHAIN: polkadot_core_primitives::Balance =
-	NATIVE_FEE_WHEN_TRANSFER_TO_PARACHAIN / 20;
+const BASE_FEE_WHEN_TRANSFER_NON_NATIVE_ASSET: polkadot_core_primitives::Balance = 5000000000;
+const KSM_FEE_WHEN_TRANSFER_TO_PARACHAIN: polkadot_core_primitives::Balance = BASE_FEE_WHEN_TRANSFER_NON_NATIVE_ASSET/ 20;
 const USDT_FEE_WHEN_TRANSFER_TO_PARACHAIN: polkadot_core_primitives::Balance =
-	NATIVE_FEE_WHEN_TRANSFER_TO_PARACHAIN / 10;
+	BASE_FEE_WHEN_TRANSFER_NON_NATIVE_ASSET / 10;
 
-decl_test_relay_chain! {
-	pub struct KusamaRelay {
-		Runtime = kusama_runtime::Runtime,
-		XcmConfig = kusama_runtime::xcm_config::XcmConfig,
-		new_ext = kusama_relay_ext(),
-	}
+decl_test_relay_chains! {
+	#[api_version(5)]
```

Comment:
How do we know it's `5`?

Reply:
Yeah, I think the [latest api version is 11](https://github.com/paritytech/polkadot-sdk/blob/f6eeca91b6cf0810ca3d8e7ea23988d9510851ba/polkadot/primitives/src/runtime_api.rs#L296). But for `release-polkadot-v1.1.0`, I think [version 5 is enough](https://github.com/paritytech/polkadot-sdk/blob/release-polkadot-v1.1.0/polkadot/primitives/src/runtime_api.rs#L129).

Reply:
Also, I tried using `11` but it did not work...

Reply:
Don't change it. As of `release-polkadot-v1.1.0`, it's only up to version 5.

Reply:
I see, thanks for pointing me to it 👍